packit.yaml

```
@@ -14,6 +15,6 @@ jobs:
     trigger: pull_request
     metadata:
       targets:
-      # once this works, move to fedora-all
       - fedora-33
       - fedora-34
+      - centos-stream-x86_64
```

```suggestion
      - centos-stream-8-x86_64
```

to make this explicit for 8; the target you provided won't be working in future since won't be clear if you want 8 or 9 :)

---

Thanks! Applied. (And indeed in the  future I really want 9 as well :grin: )